Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,8 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+function toUpperCase(string){
```

Comment:
I think toUpperCase is a bit confusing as there's already a method called this! Can you think of a clearer name?